src/effects.js

```diff
@@ -646,12 +657,12 @@ jQuery.each( {
 } );
 
 jQuery.timers = [];
-jQuery.fx.tick = function() {
+jQuery.fx.tick = function( timestamp ) {
 	var timer,
 		i = 0,
 		timers = jQuery.timers;
 
-	fxNow = jQuery.now();
+	fxNow = timestamp || getTimestamp();
```

Comment:
Just removing this line (& the one setting `fxNow` to `undefined` at the end) makes Chrome fail most of effects tests.